app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```

Comment:
@jw9billions if you wanted to set up confirmation emails, you would do it like this: 

https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users